### PR TITLE
[festival]:completed for all screens

### PR DESCRIPTION
**Pull request by dkunf**
mano bandymas padaryti festivalio saito isvaizda
jis yra ./projects/flex/Festival Site Design/index.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -173,37 +180,35 @@ naujos eilutės centre*/
...
-  right: 0;
-  top: 5rem;
-  width: 40%;
-  max-height: 80%;
-  border: 2px dashed #efefef;
-  overflow-y: scroll;
-  padding: 1rem;
-  box-shadow: var(--shadows);
+/* links to tasks */
+#link-to-tasks {
```
Comment (pavelVCS) [inline style.css:207]: its better to do not use id for styling

File: style.css [hunk 2]
```diff
@@ -173,37 +180,35 @@ naujos eilutės centre*/
...
-.pamokslas:before {
-  content: "pamokslo pradžia";
-  position: relative;
-  left: -2em;
-  background-color: rgb(211, 127, 58);
-  transform: rotate(-2deg);
-  box-shadow: var(--shadows);
-  font-size: 0.7em;
-  padding: 0.3rem;
+#contents-box {
```
Comment (pavelVCS) [inline style.css:183]: better to do not use id for styling

**Review by pavelVCS — APPROVED** [review]
good job!

**Comment by dkunf** [comment]
ačiū

**Pull request merged**